client/src/document/index.scss

```diff
@@ -450,6 +450,7 @@ code,
   border-radius: var(--elem-radius);
   font-family: var(--font-code);
   font-size: var(--type-smaller-font-size);
+  tab-size: 4;
```

Comment:
Am I right to assume we don't use tabs in code samples for other languages, so this shouldn't affect any existing code samples?

Reply:
I could only find one occurrence here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#using_the_space_parameter

using:

```bash
grep -ri --binary-files=without-match --exclude="./files/en-us/_redirects.txt" " " ./files
```